### PR TITLE
Add 3rd gen SIW content

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Contains 3rd Gen SIW content.
- **Is this PR related to a Monolith release?** Yep. 2023.04.0

### Resolves:

* [OKTA-588280](https://oktainc.atlassian.net/browse/OKTA-588280)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 1]
```diff
@@ -53,17 +53,25 @@ The widget upgrade for a redirect sign-in flow depends on whether you configured
...
+  - If you're using the [custom domain feature](/docs/guides/custom-url-domain/) and the version isn't correct, you can pin the widget's version. Click **Edit** in the **Sign-In Widget Version** section and then select the **Version** field.
+  - (EA users: You can't customize the Sign-In Widget for the Okta default brand.)
 
-4. Test your app's supported authentication sign-up and recovery flows to ensure that they still work.
+  > **Note:** When the third generation of the Sign-In Widget is enabled, the pinned version is ignored.
 
-5. Verify the CSS and localization changes that you made are reflected in the new version.
+1. Click **Save to draft**.
+
+1. Test your app's supported authentication sign-up and recovery flows to ensure that they still work.
```
Comment (gormanho-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md]: How is this done? As far as I know, we only support previewing the draft sign-in page but I don't think you can actually go through the flows. Is it a feature of the new SIW?

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md]: Good point. I'll work with you to update this step.

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md]: Updated.

**Review by gormanho-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 2]
```diff
@@ -191,19 +206,19 @@ function success(res) {
...
 
 * `features.rememberMe`: Displayed the "Remember me" function when a user signs in. See the [Organization Settings](https://help.okta.com/okta_help.htm?id=ext_Security_General) section in the Okta product documentation to enable this feature.
 
 * `features.autoPush`: Displayed a checkbox to enable the "Send push automatically" function in the MFA challenge flow. It may be added to the policy configuration in a future release.
 
-* `features.smsRecovery`: Recovered the password for users with a configured mobile phone number by using an SMS message. See the [password recovery policy](https://help.okta.com/okta_help.htm?id=ext-add-self-service-password-reset) topic in the Okta product documentation to enable and configure a possession (for example, a phone) authenticator.
+* `features.smsRecovery`: Recovered the password for users with a configured mobile phone number by using an SMS message. See  [password recovery policy](https://help.okta.com/okta_help.htm?id=ext-add-self-service-password-reset) to enable and configure a possession (for example, a phone) authenticator.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:217]:
```suggestion
* `features.smsRecovery`: Recovered the password for users with a configured mobile phone number by using an SMS message. See [password recovery policy](https://help.okta.com/okta_help.htm?id=ext-add-self-service-password-reset) to enable and configure a possession (for example, a phone) authenticator.
```

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Pull request merged**